client/codegen_wrapped/src/type_info.rs

```diff
@@ -156,7 +156,7 @@ type_info!(
     "Teleportal.Client.Object.ObjectHandle",
     false,
     "Teleportal.Client.Object.ObjectHandle",
-    O::Owned,
+    O::Ref,
```

Comment:
We support both semantics not just ref
```suggestion
    O::Both,
```